Podfile.lock

```diff
@@ -194,6 +195,6 @@ SPEC CHECKSUMS:
   zxcvbn-ios: fef98b7c80f1512ff0eec47ac1fa399fc00f7e3c
   ZXingObjC: 8898711ab495761b2dbbdec76d90164a6d7e14c5
 
-PODFILE CHECKSUM: 50ea16fe614e202ee198f2378d85dca5b2cf21a8
+PODFILE CHECKSUM: 24c4218f1b6fef3b8605b4af17ed3b914f373774
 
-COCOAPODS: 1.15.2
+COCOAPODS: 1.16.2
```

Comment:
This version doesn't match the one in the Gemfile. We should `bundle update cocoapods` to keep them in sync and only ever run `bundle exec pod …` to make sure that we're using the one specified in the Gemfile.

IIRC, we also need to keep this in sync across the SDK, App and release tools (sorry!)